src/main/java/com/spartronics4915/frc2020/RobotContainer.java

```diff
@@ -41,56 +41,88 @@ public AutoMode(String name, Command command)
 
     public final AutoMode[] mAutoModes;
 
-    private Joystick mJoystick = new Joystick(Constants.OI.kJoystickId);
-    private Joystick mButtonBoard = new Joystick(Constants.OI.kButtonBoardId);
+    private final Climber mClimber;
+    private final Intake mIntake;
+    private final Launcher mLauncher;
+    private final PanelRotator mPanelRotator;
+    private final ClimberCommands mClimberCommands;
+    private final PanelRotatorCommands mPanelRotatorCommands;
+
+    private final Joystick mJoystick;
+    private final Joystick mButtonBoard;
 
     /**
      * The container for the robot. Contains subsystems, OI devices, and commands.
      */
     public RobotContainer()
     {
+        mClimber = new Climber();
+        mIntake = new Intake();
+        mLauncher = new Launcher();
+        mPanelRotator = new PanelRotator();
+        mClimberCommands = new ClimberCommands(mClimber);
```

Comment:
This code builds, but passing the subsystem to both the containing class and the Command seems redundant.